### PR TITLE
[CHORE] adding contributing guide

### PR DESCRIPTION
**Pull request by nicolastakashi**
#212 - Adding contributing guide

### CODE REVIEW

File: CONTRIBUTING.md
Comment (nicolastakashi) [inline CONTRIBUTING.md:60]: We don't have a conclusion about that.

Should we move without this process, or should we sit together to align minimal requirements?

File: CONTRIBUTING.md
Comment (nicolastakashi) [inline CONTRIBUTING.md:64]: I would love to have one, but I think it could be another issue.

File: CONTRIBUTING.md
Comment (nicolastakashi) [inline CONTRIBUTING.md:101]: Same as PR Lifecycle.

**Comment by oded-dd** [comment]
I would go ahead with what we have and add it as we go

File: CONTRIBUTING.md
Comment (ArthurSens) [reply CONTRIBUTING.md:64]: It would be nice to provide Codespaces/Gitpod setup here :)

File: CONTRIBUTING.md
Comment (ArthurSens) [reply CONTRIBUTING.md]: We already have, don't we?

You can see it in action on my first PR: https://github.com/coralogix/telemetry-shippers/pull/195

File: CONTRIBUTING.md
Comment (matej-g) [inline CONTRIBUTING.md:54]:
```suggestion
The best way to reach us with a question when contributing is to ask on the original GitHub issue.
```

File: CONTRIBUTING.md
Comment (matej-g) [inline CONTRIBUTING.md:47]: I think we want to remove these boilerplate texts in cursive?

Comment (matej-g) [reply CONTRIBUTING.md:47]: I think we could even remove this whole paragraph, I don't think for now there's that much stuff to work on and we don't have any public channels (like Slack) where people could reach out.

File: CONTRIBUTING.md
Comment (matej-g) [reply CONTRIBUTING.md:64]: +1 to @ArthurSens, that would be a nice start, eventually we could make it possible to give local setup with e.g. KinD (it probably would not be too different from setting up E2E tests).

File: CONTRIBUTING.md
Comment (matej-g) [reply CONTRIBUTING.md:60]: Maybe for now we could just tell people to:
- Open a PR; if it's not finished yet, please make it a draft first
- Reviewers will be automatically assigned based on code owners (TO-DO)
- Reviewers will get to the PR as soon as possible, but usually within X days
- Conditions for PR to be merged (e.g. CLA, passing CI, resolved discussions)

File: CONTRIBUTING.md
Comment (matej-g) [inline CONTRIBUTING.md:70]: Let's specify, I think we already use CLA?

File: CONTRIBUTING.md
Comment (matej-g) [reply CONTRIBUTING.md:109]: See my other comment, we could keep this simple for now.

**Review by matej-g — COMMENTED** [review]
I think it looks good for now. As we don't have too much activity from outside of the company yet, I think we can have a kind of minimal version and move from there. I'd just remove any boilerplate texts that still remain.

File: CONTRIBUTING.md
Comment (nicolastakashi) [reply CONTRIBUTING.md]: Great!
Changed, can you check it?

File: CONTRIBUTING.md
Comment (nicolastakashi) [reply CONTRIBUTING.md]: Oh!
I didn't remember it, thanks.
I changed can you look again?

File: CONTRIBUTING.md
Comment (ArthurSens) [inline CONTRIBUTING.md:70]: Do we use DCO? I'm not familiar with the difference between DCO and CLA to be honest 😅

File: CONTRIBUTING.md
Comment (ArthurSens) [inline CONTRIBUTING.md:117]: We probably need to set up a required CI check to make sure we don't merge PRs without CLA signed

File: CONTRIBUTING.md
Comment (ArthurSens) [inline CONTRIBUTING.md:64]: Here I'd explain that you need at least helm installed, and redirect to a page that explains how to install helm

**Review by ArthurSens — COMMENTED** [review]
Looks good!

We probably want to remove the remaining boiler plaiting before merge though 😛

File: CONTRIBUTING.md
Comment (matej-g) [inline CONTRIBUTING.md:68]:
```suggestion
```

File: CONTRIBUTING.md
Comment (matej-g) [inline CONTRIBUTING.md:105]:
```suggestion
```

**Review by matej-g — COMMENTED** [review]
Looks good! Let's remove last leftovers and this can go 👍

File: CONTRIBUTING.md
Comment (nicolastakashi) [reply CONTRIBUTING.md:70]: DCO is basically the requirement of signed commits

File: CONTRIBUTING.md
Comment (nicolastakashi) [reply CONTRIBUTING.md:117]: Yup!
We already have the DCO bot, but not the CLA yet I think so.

File: CONTRIBUTING.md
Comment (nicolastakashi) [reply CONTRIBUTING.md]: Done!
Can you check it?

**Review by ArthurSens — APPROVED** [review]
Let's just remove the boiler plate and LGTM 🚀 

Running `make fmt-docs` should fix CI

File: CONTRIBUTING.md
Comment (ArthurSens) [inline CONTRIBUTING.md:54]:
```suggestion
```

File: CONTRIBUTING.md
Comment (ArthurSens) [inline CONTRIBUTING.md:66]:
```suggestion
```

File: CONTRIBUTING.md
Comment (matej-g) [inline CONTRIBUTING.md:66]: Wait now I'm confused, I though we use CLA, not DCO? I think we need to require only one.

**Review by matej-g — COMMENTED** [review]
Last thingy

File: CONTRIBUTING.md
Comment (nicolastakashi) [reply CONTRIBUTING.md:66]: Should I keep only CLA so?

Comment (ArthurSens) [reply CONTRIBUTING.md:66]: Looks like we do have DCO though, I'm confused 🤔 

image

Comment (nicolastakashi) [reply CONTRIBUTING.md:66]: DCO was added by me by mistake!
I'm removing the bot and also removing from document.

**Pull request merged**
